src/change_stream/mod.rs

```diff
@@ -107,6 +111,27 @@ pub struct ChangeStream {
 }
 
 impl ChangeStream {
+    pub(crate) fn new(
+        change_streams_spec: ChangeStreamSpecification,
```

Comment:
nit: it's a little redundant to call this a `change_streams_spec`, can we just call it `spec`?

Reply:
fixed